app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,172 @@
+nav {
+  border-radius: 3%;
+  height: 5em;
+}
+
+.medium {
+  height: 45em;
```

Comment:
Properties should be ordered background, background-size, display, height, width